Fonts setup

pettinarip opened the pull request:
## Description

Adds the DS fonts.

pettinarip committed:
add fonts

nhsz commented on src/pages/_app.tsx:
```diff
@@ -2,15 +2,41 @@ import type { AppProps } from "next/app"
 // Chakra custom theme
 import theme from "@/@chakra-ui/theme"
 
+const inter = Inter({
```
we could move `inter` and `mono` declarations into `@lib/fonts` to clean `_app` a bit

pettinarip replied:
Agreed.

nhsz reviewed (changes requested):
asking for a small change, looks good!

pettinarip committed:
refactor

pettinarip committed:
remove display swap since its the default value + remove weight from inter to use the variable font type

pettinarip commented:
@nhsz check the last commit, I removed the `display` value since the default is already `swap`. And removed the `weight` attr from the Inter font to make use of the variable font version of it (which is faster since we need to download only one file/font). The IBM font doesn't offer a variable font type so, in that case, we are forced to specify the weight.

nhsz committed:
Merge branch 'dev' into fonts

nhsz reviewed (approved):
LGTM!
